src/test/java/nurseybook/logic/commands/DeleteTaskCommandTest.java

```diff
@@ -1,29 +1,27 @@
-package seedu.address.logic.commands;
+package nurseybook.logic.commands;
```

Comment:
Missing imports in this file as well that would be better to include at the start!